Update file-upload.md (#1346)
api-platform/docs: core/file-upload.md

  "contentUrl": "<url>"
}
```

> ## Accessing Your Media Objects Directly 

Reviewer: ```suggestion

## Accessing Your Media Objects Directly 
```

}
```

> ## Accessing Your Media Objects Directly 

Reviewer: ```suggestion
## Accessing Your Media Objects Directly
```

> You will need to modify your Caddyfile to allow the above `contentUrl` to be accessed directly. If you followed the above configuration for VICH Uploader, that will be in `api/public/media`. Add your folder to the list of path matches, eg `|^/media/|`:

Reviewer: ```suggestion
You will need to modify your Caddyfile to allow the above `contentUrl` to be accessed directly. If you followed the above configuration for the VichUploaderBundle, that will be in `api/public/media`. Add your folder to the list of path matches, e.g. `|^/media/|`:
```

> ```

Reviewer: ```suggestion
```caddyfile
```
